.github/workflows/create-pr.yaml

```
@@ -0,0 +1,58 @@
+        with:
+          node-version: "20"
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 9.5.0
+          run_install: false
+
+      - name: Get pnpm directory
+        shell: bash
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:50: Double quote to prevent globbing and word splitting

(shellcheck)